src/index.html

```diff
@@ -7,12 +7,54 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="stylesheet"
+      href="/src/styles/index.scss"
     />
   </head>
+
   <body>
     <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="MacBook"
+        />
+      </div>
+      <p class="card__text">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__kod">Product code: 195434</p>
+      <div class="card__review-wrapper">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__review">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-label">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+      <button
+        data-qa="hover"
+        class="card__button"
+      >
+        BUY
+      </button>
+    </div>
   </body>
 </html>
+w
```

Comment:
There is a stray 'w' character outside of the HTML tags. This should be removed as it may cause unexpected display issues or errors.